etc/kayobe/apt.yml

```diff
@@ -61,8 +61,13 @@ stackhpc_apt_repositories:
     components: upload
     architecture: amd64
     trusted: yes
-  - url: "{{ stackhpc_repo_docker_ce_ubuntu_url }}"
-    suites: "{{ ansible_facts.distribution_release }}"
+  - url: "{{ stackhpc_repo_docker_ce_ubuntu_focal_url }}"
```

Comment:
Should this use the same URL as the first 2 repos (L51)? Similar for suites. We don't want both repos enabled.

Reply:
Done